yaml.yaml

```
@@ -0,0 +1,11 @@
+database_password: "hardcodedpassword"
+max_connections: "100"
+server:
+  host: "localhost"
+  port: 8080
+log_level: "info"
+log_level: "debug"
```

yamllint

[error] duplication of key "log_level" in mapping

(key-duplicates)